telemetry/definitions/commonDefinitions.json

```diff
@@ -251,7 +251,7 @@
         {
             "name": "codeTransformApiNames",
             "type": "string",
-            "description": "Names of allowed API calls",
+            "description": "Deprecated",
```

Comment:
please keep the description as it and add + deprecated. 
so that it's easy to lookup existing metrics.

Reply:
I went with the suggestion from @justinmk3 to keep description clear by leading with deprecated. I can add some original descriptions back for tracking purpose. 
```
"description": "Deprecated (Names of allowed API calls)"
```